src/cascadia/CascadiaPackage/Package-Dev.appxmanifest

```diff
@@ -23,6 +25,14 @@
     <DisplayName>ms-resource:AppStoreNameDev</DisplayName>
     <PublisherDisplayName>A Lone Developer</PublisherDisplayName>
     <Logo>Images\StoreLogo.png</Logo>
+    <!-- Older versions of Windows 10 respect this -->
+    <desktop6:RegistryWriteVirtualization>disabled</desktop6:RegistryWriteVirtualization>

```

Comment:
Oh, and one more question... This disables HKCU virtualization altogether. Will this conflict with any registry keys written by MSIX/UWP itself? (Like those written to register the application with the OS.)

Reply:
HKCU virtualization has always been enabled for us, this actually doesn't change anything.

That's why we had the horrible reg.exe workaround before.

It only applies to registry writes performed from within the package, not the ones during registration.